schema/spacewalk/postgres/packages/rpm.pkb

```diff
@@ -69,38 +65,6 @@ $$ language 'plpgsql';
         then
             return 0;
         end if;
-        if POSITION('+' in str1) <> 0 AND POSITION('+' in str2) <> 0 AND POSITION('~' in str1) = 0 AND POSITION('~' in str2) = 0 AND POSITION('.module' in str1) = 0 AND POSITION('.module' in str2) = 0
```

Comment:
I think not. `~` and `^` are also used in rpm. In fact RPM stole them from debian.

Reply:
'~' is allowed in package version for rpm:

```
susemanager=# select distinct(ch.name) from rhnpackage p inner join rhnpackageevr evr on p.evr_id = evr.id inner join rhnpackagename name on p.name_id = name.id inner join rhnchannelpackage c on p.id = c.package_id inner join rhnchannel ch on c.channel_id = ch.id where evr.version like '%~%' and ch.name like 'SLE%Server%';
                            name                             
-------------------------------------------------------------
 SLE-Module-Server-Applications15-SP1-Pool for x86_64
 SLE-Module-Server-Applications15-SP1-Pool for x86_64 SAP
 SLE-Module-Server-Applications15-SP1-Updates for x86_64
 SLE-Module-Server-Applications15-SP1-Updates for x86_64 SAP
 SLE-Module-Server-Applications15-SP2-Pool for x86_64
 SLE-Module-Server-Applications15-SP2-Updates for x86_64
 SLE-Module-Server-Applications15-Updates for x86_64
(7 rows)
```

Reply:
Both `~` and `^` are valid in rpm versions and have special meaning for sorting. I know at least `~` is used but `^` was new to me and since my version parser checks for valid chars and i have not encountered any error on suse repos so far i guess that means its not widely used.